task solution

VladaProskura opened the pull request:
- [DEMO LINK](https://vladaproskura.github.io/react_todo-app-with-api/)

VladaProskura committed:
solution

task solution

etojeDenys commented on src/App.tsx:
```diff
@@ -1,26 +1,256 @@
+      .map(todo => todo.id);
+
+    completedIds.forEach(id => handleDeleteTodo(id));
+  };
```
it is better to use Promise.all here

etojeDenys reviewed (changes requested):
1. toggle all button should look different
your:
<img width="606" height="358" alt="image" src="https://github.com/user-attachments/assets/943eab74-d024-482f-bb80-6d4fa2709a54" />

example:
<img width="671" height="294" alt="image" src="https://github.com/user-attachments/assets/8613ac4c-0664-44c5-bf00-2db231981e01" />

2. after successfully adding a post, the input should become focused

3. the editing style is different
your: 
<img width="543" height="239" alt="image" src="https://github.com/user-attachments/assets/d32f6521-3730-416d-be65-7f617f7b9ca2" />

example:
<img width="582" height="406" alt="image" src="https://github.com/user-attachments/assets/ac112972-f383-4821-aa93-2072b276b481" />

4. do not forget to implement these steps
<img width="691" height="398" alt="image" src="https://github.com/user-attachments/assets/a3d4b128-ccdb-45f9-9699-ac0bbb326209" />

VladaProskura committed:
expected to find element: [data-cy=TodoTitleField, but never found it.]

expected find element: [data-cy=TodoTitleField, but never found it.]

does not work Enter in the edit form

VladaProskura committed:
task solution

etojeDenys reviewed (changes requested):
1. after pressing the toggle buttons we see this error. however, when I try to change the status, it changes
<img width="555" height="529" alt="image" src="https://github.com/user-attachments/assets/1cd2ea06-89c3-4882-8822-1b6119ff9ac6" />

2. position of the toggle button is incorrect
example: 
<img width="417" height="68" alt="image" src="https://github.com/user-attachments/assets/88c3414a-3009-40fe-a647-ef0546461f6c" />

your: 
<img width="611" height="357" alt="image" src="https://github.com/user-attachments/assets/8c3f4a36-6813-4cb6-aa3b-eb72971d95a7" />

3. the same problem with editing style
<img width="684" height="729" alt="image" src="https://github.com/user-attachments/assets/fb026537-1965-49ec-b619-9166c23b399c" />

VladaProskura committed:
final solution

need help with: "should not toggle a todo"

Anton-Kuchmasov reviewed (changes requested):
Nice work you did!

Please fix the styles here:
<img width="715" height="550" alt="image" src="https://github.com/user-attachments/assets/10211d2e-8966-4b00-830e-614b3a252108" />

(check out an initial markup - it might help you find correct className for the input)

VladaProskura committed:
solution

etojeDenys reviewed (approved):
lgtm
